desktop/cmp/panel/impl/dragger/DraggerModel.js

```diff
@@ -23,6 +26,21 @@ export class DraggerModel {
     constructor(panelModel) {
         this.panelModel = panelModel;
         this.throttledSetSize = throttle(size => panelModel.setSize(size), 50);
+
+        // We use a ref to allow us to add listeners directly to the element. This allows
+        // us to mark events as non-passive, which in turn allows us to preventDefault()
+        this.addReaction({
+            track: () => this.ref.current,
+            run: (current) => {
+                if (current) this.addListeners(current);
+            }
+        });
+    }
+
+    addListeners(el) {
+        el.addEventListener(XH.isDesktop ? 'dragstart' : 'touchstart', e => this.onDragStart(e));
```

Comment:
Are we sure that the passive false was needed here, or is it just enough to forego the react "synthetic" events?

Just trying to be careful on this one here, as the change currently effects all of our code base here in order to fix the Airwatch problem we are seeing at a client

Reply:
Yes - most browsers default `touchmove` (and scroll events) to passive in order to improve performance. Without setting `{passive: false}`, the call to preventDefault() is ignored and you get warnings in the console.